migration/article/Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM ruby:2.7.5-alpine
+FROM alpine:3.15
 
-RUN mkdir /db
-COPY Gemfile /db
-COPY Gemfile.lock /db
-WORKDIR /db
+WORKDIR /app
 
-RUN apk add --no-cache build-base mariadb-dev && \
-    gem install bundler:2.2.13 && \
-    bundle install
+COPY schema.sql .
 
-ENTRYPOINT ["rake"]
+RUN apk add --no-cache curl tar && \
```

Comment:
⚠️ **[hadolint]** <[DL3018](https://github.com/hadolint/hadolint/wiki/DL3018)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`